server/src/main/java/org/elasticsearch/index/mapper/KeywordFieldMapper.java

```diff
@@ -1085,15 +1099,6 @@ public Query automatonQuery(
         ) {
             return new AutomatonQueryWithDescription(new Term(name()), automatonSupplier.get(), description);
         }
-
-        /**
-         * The name used to store "original" that have been ignored
-         * by {@link KeywordFieldType#ignoreAbove()} so that they can be rebuilt
-         * for synthetic source.
-         */
-        public String originalName() {
-            return originalName;
```

Comment:
this has been moved up to `TextFamilyFieldType`, which `KeywordFieldType` now extends